src/Sites/twitch.tv/Runtime/RecentMessagesProvider.ts

```diff
@@ -0,0 +1,98 @@
+import { Twitch } from 'src/Sites/twitch.tv/Util/Twitch';
+
+const RECENT_MESSAGES_API_URL = 'https://recent-messages.robotty.de/api/v2';
+
+export class RecentMessagesProvider {
```

Comment:
As it stands only actual chat messages are supported. The API also returns messages like sub notifications, but I based this PR on how historical messages are loaded when you're the moderator of a channel. The chat history that mods see on page load doesn't include those kinds of messages, so I didn't bother with adding support for that. But I don't think it would be too difficult to do so.